test/client_service_base.test.js

```diff
@@ -569,6 +569,7 @@ describe('executeContract', () => {
       setSignature: function() {},
       setUseFunctionIds: function() {},
       setFunctionIdsList: function() {},
+      setNonce: function() {},
```

Comment:
This part is to make the original unit-test work.
Don't worry about the test. It will be handled later.